pkg/pointerdb/pointerdb_test.go

```diff
@@ -97,6 +99,14 @@ func TestServiceGet(t *testing.T) {
 
 	validAPIKey := console.APIKey{}
 	apiKeys := &mockAPIKeys{}
+	// creating in-memory db and opening connection
+	satdb, err := satellitedb.NewInMemory()
```

Comment:
we're moving toward `satellitedbtest.Run()` for anything like this, to test against both Sqlite and Postgres

Reply:
I thought about it. This is my understanding. Make that change will make the test case run twice exactly the same. It doesn't add much value. All I needed was to satisfy allocation.go's payerbandwidthfunc(), which needed a pointer to a DB ( any db).

Reply:
SQLite handles dates differently than Postgres, which has caused errors which only show up in non-GMT time zones when running on Postgres.  Travis runs as GMT, so these issues don't show up in Travis.  Sometimes its nice to run the test twice. :) :P

Reply:
I'll concede that this is not strictly necessary for this PR, but you should get in the habit of thinking of satellitedbtest.Run() as a _good thing_.